slicer-build-base/update.py

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env python
+
+import argparse
+import os
+import subprocess
+
+#
+# Usage:
+#
+#   $ ./update.py <svn_revision> <svn_revision_date>
+#
+# Example:
+#
+#   $ ./update.py 25199 2016-06-19
+#
+
+def update_dockerfile(dockerfile, svn_revison, svn_revision_date):
+    lines = []
+    with open(dockerfile) as f:
+        for line in f:
+            if 'ENV SLICER_VERSION' in line:
+                line = "ENV SLICER_VERSION %s\n" % svn_revison
+            if '# Slicer master ' in line:
+                line = "# Slicer master %s\n" % svn_revision_date
+            lines.append(line)
+    with open(dockerfile, 'w') as f:
+        f.writelines(lines)
+        print("File %s updated\n" % dockerfile)
+
+def _run(cmd):
+    print("Executing %s" % ' '.join(cmd))
+    subprocess.Popen(cmd)
```

Comment:
This returns immediately. `subprocess.check_call()` would be a good alternative.